vuepress/docs/next/tutorials/devops/plugin-configuration.md

```diff
@@ -0,0 +1,82 @@
+---
+sidebarDepth: 2
+---
+
+# Customize Deployment with Plugin Configuration Profiles
+
+This tutorial describes three methods to utilize configuration profiles in the OperatorConfigMap to specify resource allocation. This provides a simple way to customize the deployment parameters of Entando bundle plugins for improved efficiency.
```

Comment:
rework first sentence: current structure is "to do ___ in ___ to ___" --> change to "to do ___ in ___"
e.g. "This tutorial describes three methods to specify resource allocation using configuration profiles in the `OperatorConfigMap`." 
also add backticks to `OperatorConfigMap`

is the object a plugin belonging to an Entando Bundle or a bundle plugin created with Entando? (to resolve capitalization of "bundle")

Reply:
I think we agreed to simplify this to "This tutorial describes three methods to utilize configuration profiles to specify resource allocation for Entando plugins." And then I'd make the 2nd sentence "This provides a simple way to customize the plugin deployment parameters for improved efficiency."

For the future, the only kind of bundles in Entando are Entando Bundles. And a plugin could be generated or hand-crafted, it's the component type used to deploy microservices.